Dockerfile

```
@@ -0,0 +1,11 @@
+FROM python:3.10.5-slim-buster
+
+COPY Pipfile /app/
+COPY Pipfile.lock /app/
+WORKDIR /app
+RUN pip3 install pipenv && pipenv install --deploy --clear --system;
+COPY .env /app/
```

Use full potential of `WORKDIR` 
```suggestion
WORKDIR /app
COPY Pipfile .
COPY Pipfile.lock .
RUN pip3 install pipenv && pipenv install --deploy --clear --system;
COPY .env .
```